INSTALL.md

```diff
@@ -16,8 +16,8 @@ Use the terminal, Luke:
 
 1. download or clone this repository
 2. extract (if downloaded compressed)
-3. move the extracted folder into your Steam skin folder (which is in the Steam base folder)
-   default location: C: -> Program Files -> Steam -> Skins
+3. move the extracted folder into your Steam skin folder (which is in the Steam base folder)  
+   default location: `C: -> Program Files -> Steam -> Skins`
```

Comment:
we might want to write file paths as, well, file paths :) e.g. `C:/Program Files/Steam/Skins`